padd.sh

```diff
@@ -87,16 +83,13 @@ mega_status_unknown="${check_box_question} Unable to determine Pi-hole status."
 # TINY STATUS
 tiny_status_ok="${check_box_good} System is healthy."
 tiny_status_update="${check_box_info} Updates are available."
-tiny_status_hot="${check_box_bad} System is hot!"
 tiny_status_off="${check_box_bad} Pi-hole is offline"
 tiny_status_ftl_down="${check_box_info} FTL is down!"
 tiny_status_dns_down="${check_box_bad} DNS is off!"
 tiny_status_unknown="${check_box_question} Status unknown!"
 
 # Text only "logos"
 padd_text="${green_text}${bold_text}PADD${reset_text}"
```

Comment:
Why did retro get removed?

Reply:
Because of these `shellcheck` warnings when enabling SC2034 and SC2154 which is the subject of this PR 😉 
```
SC2034 (warning): tiny_status_hot appears unused. Verify use (or export if used externally).
SC2034 (warning): padd_text_retro appears unused. Verify use (or export if used externally).
SC2034 (warning): mini_text_retro appears unused. Verify use (or export if used externally).
```